config/locales/it.yml

```diff
@@ -0,0 +1,741 @@
+# See https://github.com/svenfuchs/rails-i18n/tree/master/rails%2Flocale for starting points.
+
+# Convenzioni adottate:
+#   - tag → etichetta
+#   - story → articolo
+#   - merge/unmerge → accorpare/scorporare
+#   - ban → bannare
+#   - thread → discussione
+#   - author/submitter → contributore
```

Comment:
contributore suona un po' strano in italiano. Autore?

Reply:
Autore è la traduzione più scontata, ma considerando che appunto non si è autori del contenuto, ma solo contributori della sua divulgazione, ho preferito adoperare questo termine.
Non è il caso?

Reply:
Si è però autori del post. Io lo intendevo così

Reply:
> Si è però autori del post. Io lo intendevo così

È valido, ma introduce un'ambiguità omissibile utilizzando un termine più specifico (come fatto).
Introduco comunque la modifica?

Reply:
Secondo me si

Reply:
Anche secondo me.

Reply:
D'accordo.